src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSource.java

```diff
@@ -493,18 +484,6 @@ public boolean isAutoRegisterHook() {
         return true;
     }
 
-    public BitbucketRepositoryType getRepositoryType() throws IOException, InterruptedException {
-        if (repositoryType == null) {
-            BitbucketRepository r = buildBitbucketClient().getRepository();
-            repositoryType = BitbucketRepositoryType.fromString(r.getScm());
-            Map<String, List<BitbucketHref>> links = r.getLinks();
-            if (links != null && links.containsKey("clone")) {
-                cloneLinks = links.get("clone");
-            }
```

Comment:
OK, even if `cloneLinks` is not set here, it will be set before it is used.

Reply:
Yeah. Good catch. I removed more than need. Thanks.

Reply:
I reduced performed changes